### DISCUSSION TITLE
how to install rollercoaster tycoon 1?

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23114] [Q&A]
i try to install rollercoaster tycoon 1. I have installed it into my windows with 2 expansions.
Then in openrct2 I choose the folder but then it says a file csg1i.dat is missing.
What to do?

I have CD version with these cd names in windows explorer

RCTYCOON
RCTEXP1
RCTEXP2

there is no error when i install it in windows

**Comment by telk5093** [comment]
Did you install RCT1 on your PC by CD?

**Reply by ghost** [reply]
> Did you install RCT1 on your PC by CD?

Yes! I have rct1 and rct2 with all expansions on Cd. Everything is installed.

**Comment by Lastorder-DC** [comment]
"The specified path contains a RollerCoaster Tycoon 1 installation, but the “csg1i.dat” file is missing. This file needs to be copied from the Loopy Landscapes or RCT Deluxe CD to the “Data” folder of the RollerCoaster Tycoon 1 install on your hard drive."

https://github.com/OpenRCT2/OpenRCT2/blob/0611c18bf65ff1fa80354d3cf9f779c71f928890/data/language/en-GB.txt#L3446

You should copy `csg1i.dat` from CD to installation folder.
